03-Custom-Actions-In-Python/06-Improving-Performance.ipynb

```diff
@@ -92,7 +92,7 @@
     "profiling tools available. However, after profiling here are some tips\n",
```

Comment:
<p>Their seems to be some weird formatting going on here. with <code>State.get_snapshot</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>